Paws4Me/Paws4Me/Display And Save Pets/View Controllers/LocalPetNameViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  LocalPetNameViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/10.
+//
+
+import UIKit
+
+class LocalPetViewController: UIViewController, UITableViewDelegate {
```

Comment:
You can move `UITableViewDelegate` down to your extension. 😄

Reply:
Will do so